docs/src/main/asciidoc/kubernetes-resources.adoc

```diff
@@ -139,3 +139,37 @@ By default the Kubernetes resources do not contain readiness and liveness probes
 
 The values of the generated probes will be determined by the configured health properties: `quarkus.smallrye-health.root-path`, `quarkus.smallrye-health.liveness-path` and `quarkus.smallrye-health.readiness-path`.
 More information about the health extension can be found in the relevant link:health-guide[guide].
+
+== Tuning the generated resources using application.properties
```

Comment:
Can we perhaps add a note that the old properties are no longer supported?

Reply:
Sorry, behind on this. Thx for the mention, @geoand. This is awesome!  Definitely a noteworthy feature. The Quarkus guide pointing to the dekorate docs is less than ideal given the property namespace differences. Worth a chat at some point.

Reply:
@iocanel the properties now match between the two guides don't they?

Reply:
Properties match, be the dekorate ones are prefixed with 'dekorate'. John has a very valid point here. Also, the dekorate site does have properties that are not applicable to Quarkus so we need to cherry pick what we need.